.travis.yml

```diff
@@ -42,7 +44,7 @@ install: # Use this to install any prerequisites or dependencies necessary to ru
   # Install dependencies for source repos
   - rosdep install --from-paths src --ignore-src --rosdistro $ROS_DISTRO -y
 before_script: # Use this to prepare your build for testing e.g. copy database configurations, environment variables, etc.
+  - export PYTHONPATH=/usr/lib/python2.7/dist-packages:$PYTHONPATH
   - source /opt/ros/$ROS_DISTRO/setup.bash  
 script: # All commands must exit with code 0 on success. Anything else is considered failure.
-  - catkin_make -j2
-
+  - catkin_make -j2 --cmake-args -Wno-dev
```

Comment:
Without ``-Wno-dev``, it generates a lots of warnings.
https://travis-ci.org/otamachan/ros_control/builds/95517583#L2152

It seems that it's because cmake version in TravisCI is 3.2.2.
I think It's better to add cmake_policy in catkin(https://github.com/ros/catkin) rather than in this repo.